### ISSUE TITLE
Store freezing KDE

### ISSUE THREAD

**Issue by Dave6871** [feren-OS/Bug-Reporting-Center#331]
Store application  it will lock freeze I can not even move it. takes about a minute to sync.  once it loads it functions as normal

To reproduce it close it restart and refresh the list.

Linux Mint has updated the software app in LM 22 it actually loads much faster than its has in past releases.

**Comment by dominichayesferen** [comment]
Yeah the new Store. provided I don't delay it at this point to 10th Anniversary release, already plans to fix this by having the program get everything together in the background on login (since it's also going to be the updates manager so needs to keep a constant tab on update availability).

**Comment by dominichayesferen** [comment]
Does the rebased Legacy Store have this issue still in Feren OS 2025.03?
